### PR TITLE
Admin role restricted in what it is allowed to do

### PR DESCRIPTION
**Pull request by hwindsor**
## Pull Request Overview

Admin role restricted to reflect what it is allowed to do according to the roles spreadsheet

PR 37 should be taken before this one as the Project tests will need rework once that  is taken
This currently also depends on confirmation by Tanya of some points

---

**Pull Request Checklist**
- [x] Ensure that the PR is properly formatted
  - Example: All lint checks are passing
- [x] Ensure that the PR is properly rebased
  - Example: The PR is rebased on `develop` (commit: `<insert develop commit hash>`)
- [x] Ensure that the PR uses a consolidated database migration
  - Example: One database migration is proposed (revision `<insert develop revision>` -> `<insert new revision>`)
- [x] Ensure that the PR is properly tested
  - Example: All automated tests are passing
- [x] Ensure that the PR is properly sanitized
  - Example: No sensitive data or large content was added to this PR
- [ ] Ensure that the PR is properly reviewed

### CODE REVIEW

File: tests/modules/users/test_permissions.py [hunk 1]
```diff
@@ -264,20 +264,35 @@ def test_ObjectAccessPermission_authenticated_user(authenticated_user_login):
...
-    # Admin user should be able to do what they like
-    validate_can_read_object(obj)
-    validate_can_write_object(obj)
-    validate_can_delete_object(obj)
 
-    obj.is_public = lambda: True
-    validate_can_read_object(obj)
-    validate_can_write_object(obj)
-    validate_can_delete_object(obj)
+    # obj = Mock()
```
Comment (hwindsor) [inline tests/modules/users/test_permissions.py:272]: That was a mistake, will restore

**Comment by bluemellophone** [comment]
There is still a disconnect happening here because this PR includes commits that are already in develop and are being duplicated.
One way to see this is to inspect the commits with `gitk`, some other GUI application (e.g. Sublime Merge), or by using the command below:
```
git log --color --graph --pretty=format:'%Cred%h%Creset -%C(yellow)%d%Creset %s %Cgreen(%cr)%C(bold blue)<%an>%Creset' --abbrev-commit
```

Here is a side-by-side example:

**This PR**
Screen Shot 2021-02-10 at 11 16 20 AM

**Develop branch**
Screen Shot 2021-02-10 at 11 31 26 AM

The thing to notice is that there is only 1 level between the commits in develop and the PRs that are being merged in.  Another thing to notice is that there are no crossing lines in the history of the commits.

**Comment by karenc** [comment]
I do something similar:

```
git log --graph --oneline --decorate origin/develop origin/admin_DEX160
```

```
*   f4bacdd (origin/admin_DEX160) another rebase
|\  
| *   4412517 continued rebase
| |\  
| | *   e876617 repeat merge from pull
| | |\  
| | | * ff0eecc restored tests
| | | * a3a01a9 Admin role restricted in what it is allowed to do
| | * | 0ccbd2a Updated after rebase from develop
| | * | 4ea7c89 restored tests
| | * | 8cfa526 Admin role restricted in what it is allowed to do
| * | | 8491e0b Updated after rebase from develop
| * | | 65c2544 restored tests
| * | | 045e39d Admin role restricted in what it is allowed to do
* | | | ee149cd Admin role restricted in what it is allowed to do
* | | | 77e7ea5 Updated after rebase from develop
* | | | 2e2e315 Admin role restricted in what it is allowed to do
* | | | 0f4962d Updated after rebase from develop
* | | | 6a65199 restored tests
* | | | ded0d14 Admin role restricted in what it is allowed to do
| | | | * b44e301 (origin/develop) Bump frontend version
| | | | * 9727723 Merge pull request #39 from WildMeOrg/enc_assets
| |_|_|/| 
|/| | | | 
| | | | * eefda89 per discussion with colin, del from edm when feather create/store fails
```

**Comment by karenc** [comment]
I'm going to get together with Howard tomorrow to do the rebase together

**Review by colinwkingen — APPROVED** [review]
LGTM

**Pull request merged**